pulp_2to3_migration/app/plugin/deb/pulp_2to3_models.py

```diff
@@ -0,0 +1,165 @@
+from django.db import models as django_models
+
+from pulp_2to3_migration.app.models import Pulp2to3Content
+
+from pulp_deb.app import models as pulp3_models
+
+from . import pulp2_models
+
+
+class Pulp2DebPackage(Pulp2to3Content):
+    """
+    Pulp 2to3 detail content model to store pulp 2 DebPackage content details for Pulp 3
+    content creation.
+    """
+    pulp2_type = 'deb'
+    name = django_models.CharField(max_length=255)
+    version = django_models.CharField(max_length=255)
+    architecture = django_models.CharField(max_length=255)
+    checksumtype = django_models.CharField(max_length=255)
+    checksum = django_models.CharField(max_length=255)
+    filename = django_models.CharField(max_length=255)
+    size = django_models.PositiveIntegerField()
```

Comment:
This Integer field (and/or the `installed_size` one below), appear to be causing me grief.
I keep getting `TypeError: int() argument must be a string, a bytes-like object or a number, not 'DeferredAttribute'` on the first package the pipeline tries to migrate.
When I change these fields to `django_models.TextField` (and then manually cast things back to `int` in the code), the pipeline gets a bit further (I can see some looping over all the packages I am trying to migrate) but ultimately fails with the same error.

I am somewhat at my wits end about this. What I am doing here looks similar to what rpm is doing to me...
Maybe I am using the async pipeline wrong, or maybe it is something small and trivial I am overlooking.

Reply:
I agree it's strange. Hard to suggest something without any traceback or an attempt to reproduce.
@dralley , @dkliban , any thoughts?

Reply:
@quba I left a comment down below, I don't know if it is the cause of your problem, but it seems at least plausible that it might be

Reply:
Wrong quba :) ping @quba42

Reply:
So this error appears to have gone away for now, so I am closing this thread for now.